src/core-sdk-tasks/core-sdk-tasks.csproj

```diff
@@ -10,12 +10,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Build" Version="15.7.179" />
-    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.7.179" />
+    <PackageReference Include="Microsoft.Build" Version="$(MicrosoftBuildPackageVersion)" />
```

Comment:
Bumping this all the way up to v.current makes me slightly nervous: don't we want to support older VS? Of course that mostly doesn't work because of compiler deps so maybe this is fine?

Reply:
@rainersigwald Are you saying that if I bump this to current, then any developer trying to build installer with an older VS version will be broken? Is there a recommended version I should be targeting instead (I'm also nervous about picking a version and having it be forgotten about)?  I assume that since this is just the installer build itself and it already uses a stage-0 SDK to build, I'd be fine bumping this up.

Reply:
No, building installer itself should be fine. My concern was that this might (but probably wouldn't) make the tasks built in this project fail to work on older MSBuilds (so like 8.0.2xx might fail in 17.8, even more than it already doesn't work)

Reply:
I don't think we ship any of these tasks. This is the set of tasks in the installer repo for producing the various installer packages. This builds into Microsoft.DotNet.Cli.Build which doesn't ship anywhere afaik.

Sounds like it should be safe then?

Reply:
Oh yeah then totally fine!